### PR TITLE
Added Navigation Bar 

### PR DESCRIPTION
**Pull request by JatuFaique**
Please Review
navbarr

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+.nav--items {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+.content {
+  padding: 8px 8% 8px 8%;
```
Comment (poojavishnoi) [inline components/navbar/navbar.css:53]: here you can just write 8% once and it will work for all sides

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+.nav--short {
+  width: max-content;
+  border-radius: 0 0 2rem 0;
+}
+
+.nav--items {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
```
Comment (poojavishnoi) [inline components/navbar/navbar.css:47]: Be consistent in using rem or px.(recommend to use rem)

**Review by poojavishnoi — APPROVED** [review]
Good.Just left some suggestions

File: components/navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  font-family: "Inter", sans-serif;
+  --fulvous: rgb(217, 126, 28);
+  --fulvous-light: rgba(217, 126, 28, 0.582);
+  --border-danger: rgb(216, 43, 43);
+  --bright-navy-blue: hsla(211, 77%, 48%, 1);
+  --hookers-green: hsla(152, 21%, 36%, 1);
+  --border-succes: rgb(73, 216, 149);
+  --mint-cream: hsla(120, 100%, 98%, 1);
+  --black-chocolate: hsla(57, 100%, 4%, 1);
+}
```
Comment (TejasShekar) [inline components/navbar/navbar.css:20]: If these are repeating props, move them to a seperate stylesheet and import that stylesheet here.
This way, you can keep your stylesheets specific to the component only.

File: components/navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+.nav {
+  display: flex;
+  /* position: sticky; */
```
Comment (TejasShekar) [inline components/navbar/navbar.css:24]: Remove comment

File: components/navbar/navbar.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    <link href="./navbar.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/components/avatar/avatar.css" />
+    <title>Navbar</title>
+    <script
+      src="https://kit.fontawesome.com/51b1b3d424.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="fix">
```
Comment (TejasShekar) [inline components/navbar/navbar.html:16]: Give a proper class name

**Review by TejasShekar — COMMENTED** [review]
I have left a few suggestions in the review. Have a look at them.

File: components/navbar/navbar.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="./navbar.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/components/avatar/avatar.css" />
```
Comment (TejasShekar) [inline components/navbar/navbar.html:8]: It looks like navbar is dependant upon avatar for inheriting some styles. In that case, you can use the @import tag in your navbar.css file and import the avatar.css there.

**Review by TejasShekar — COMMENTED** [review]
I have left suggestions, have a look at them.

**Pull request merged**